transports/transport-rest/transport-rest-vertx/src/main/java/io/servicecomb/transport/rest/vertx/accesslog/element/impl/CookieElement.java

```diff
@@ -0,0 +1,55 @@
+package io.servicecomb.transport.rest.vertx.accesslog.element.impl;
+
+import java.util.Set;
+
+import com.sun.org.apache.regexp.internal.RE;
```

Comment:
Please don't use the internal API which may not be compatible with other JDK.

Reply:
This is added by mistake. Has been removed.